editors/vscode/client/extension.ts

```diff
@@ -142,8 +142,7 @@ export async function activate(context: ExtensionContext) {
     synchronize: {
       // Notify the server about file config changes in the workspace
       fileEvents: [
-        workspace.createFileSystemWatcher('**/.eslintr{c,c.json}'),
-        workspace.createFileSystemWatcher('**/.oxlint{.json,rc.json,rc}'),
+        workspace.createFileSystemWatcher('**/.oxlint{.json,rc.json}'),
         workspace.createFileSystemWatcher('**/oxlint{.json,rc.json}'),
```

Comment:
probably don't need this line now?
neither the non "rc" varient wtatcher

Reply:
The user can rename the config file. That is why I kept it in :)

Reply:
The user can rename the file to anything. The workspace config should be read for the file and add a watcher here for that (I’m not sure how feasible that is).
Once that’s done, I think the other watchers can be removed.

Reply:
@nrayburn-tech agree. but this was out of scope for this PR :) 
We need to check for renaming, deletion, and creation of that file too :) 

I just removed the eslint part and `.oxlintrc` (because we force `.json`)

Reply:
imo thjs is ok for now, but definitely a TODO